qldbdriver/result.go

```diff
@@ -83,9 +127,11 @@ func (result *Result) Err() error {
 
 // BufferedResult is a cursor over a result set from a QLDB statement that is valid outside the context of a transaction.
 type BufferedResult struct {
-	values    [][]byte
-	index     int
-	ionBinary []byte
+	values            [][]byte
+	index             int
+	ionBinary         []byte
+	consumedIOs       *IOUsage
```

Comment:
When you pull timingInformation and consumedIOs into the same object you can reuse that here.

Reply:
Moved them to `statementStatistics` struct.

Reply:
Change this too

Reply:
Updated to `metrics`.